cmd/internal/apidump/apidump.go

```diff
@@ -80,6 +84,19 @@ var Cmd = &cobra.Command{
 			outFlag.AkitaURI = &uri
 		}
 
+		if postmanAPIKey != "" {
+			cfg.WritePostmanAPIKeyAndEnvironment("default", postmanAPIKey, postmanEnvironment)
+		}
+
+		// ToDo: Do we need to change akiuri package also to check proper collectionID format??
+		if postmanCollectionID != "" {
+			uri, err := akiuri.Parse(akiuri.Scheme + postmanCollectionID)
+			if err != nil {
+				return errors.Wrap(err, "bad postman collection-id name")
+			}
+			outFlag.AkitaURI = &uri
+		}
```

Comment:
No, we should further deprecate the use of the Akita URI here.  You're doing the correct thing by passing it as a separate argument to apidump.Args.  We should not need it to also be present in AkitaURI.